config-linux.md

```diff
@@ -171,14 +171,18 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
-The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` can be used to either control the cgroup hierarchy for containers or to run a new process in an existing container.
-If `cgroupsPath` is:
-* ... an absolute path (starting with `/`), the runtime MUST take the path to be relative to the cgroup mount point.
-* ... a relative path (not starting with `/`), the runtime MAY interpret the path relative to a runtime-determined location in the cgroup hierarchy.
-* ... not specified, the runtime MAY define the default cgroup path.
+### <a name="configLinuxCgroupsPath" />Cgroups Path
+
+**`cgroupsPath`** (string, OPTIONAL) path to the cgroups.
+It can be used to either control the cgroups hierarchy for containers or to run a new process in an existing container.
+
+The value of `cgroupsPath` is either an absolute path or a relative path.
```

Comment:
> empty string means the cgroupsPath is not specified…

Does the spec support that?  I think `"cgroupsPath": ""` *is* specifying `cgroupsPath`, because the `cgroupsPath` key exists in the `linux` object.  If we want to conflate `""` with “unset” for this property (or in general?), we'd need to land spec wording to that effect.

Personally, I think life is just as easy for config authors and simpler for runtimes if we forbid empty-string values where they don't make sense.  Runtimes [don't have to validate configs if they don't want][1], so making empty strings illegal would mean that runtimes could handle those configs however they like (erroring out or conflating them with whatever other case they liked).

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc5/runtime.md#L109